apps/nowcasting-app/components/play-button/index.tsx

```diff
@@ -1,18 +1,14 @@
 import React, { useRef, useState } from "react";
 import useGlobalState from "../globalState";
 import { useStopAndResetTime } from "../hooks/use-and-update-selected-time";
-import { formatISODateString } from "../utils";
+import { addMinutesToISODate, formatISODateString } from "../utils";
 import Ui from "./ui";
 
 type PlayButtonProps = {
   endTime: string;
   startTime: string;
 };
-const add30Minutes = (date: string) => {
-  var d = new Date(date);
-  d.setMinutes(d.getMinutes() + 30);
-  return d.toISOString();
-};
+
 const PlayButton: React.FC<PlayButtonProps> = ({ endTime, startTime }) => {
```

Comment:
can you auto-focus on the play button to be able to play it on click enter key (now it needs to move between page links to reach it)

Reply:
great suggestion , but i don't think it belong to this story, feel free to add to the tasks